include/verilogAST.hpp

```diff
@@ -110,6 +110,30 @@ class Identifier : public Expression {
   ~Identifier(){};
 };
 
+class Attribute : public Expression {
+ protected:
+  virtual Attribute* clone_impl() const override {
+    return new Attribute(*this);
+  };
+
+ public:
+  std::unique_ptr<Expression> value;
```

Comment:
should this be `Union[Id, Attribute]` instead of just `Expression`?

Reply:
Done